cli/pkg/secrets/secret.go

```diff
@@ -26,10 +26,15 @@ const (
 func GetOrCreateClISecret() []byte {
 	// get password
 	secret, err := keyring.Get(secretService, secretUser)
+
 	if err != nil {
 		if !errors.Is(err, keyring.ErrNotFound) {
+
+			if secretsFile := os.Getenv("ENTE_CLI_SECRETS_PATH"); secretsFile != "" {
+				return GetSecretFromSecretText(secretsFile)
+			}
```

Comment:
Thanks for the change.
Should we put this change inside the else block.
is isRunningContainser() {
} else {
   			if secretsFile := os.Getenv("ENTE_CLI_SECRETS_PATH"); secretsFile != "" {
				return GetSecretFromSecretText(secretsFile)
			}
			log.Fatal(fmt.Errorf("error getting password from keyring: %w", err))
}

Reply:
But then it only works if it is running inside a container. I want to use this env var outside of a container aswell.